stout_a05a.js

```diff
@@ -0,0 +1,29 @@
+/*Kyle Stout
+  stout_a05a
+  INFO 2124WW
+  Thoendel
+  1-19-2020
+*/
+
+let todolist = ["Eat", "Sleep", "Shower", "Brush Teeth"];
+todolist;
+todolist.unshift("Wake up");
+todolist;
+todolist.reverse();
+todolist.sort();
+todolist;
+todolist.includes("x");

```

Comment:
I'd love to see you do something with this value - i.e. you could output that your list does or does not include x.